isis/scripts/install_isis.sh

```diff
@@ -0,0 +1,355 @@
+    ISIS_VERSION=$2
+fi
+
+printf "\nBeginning install of $PACKAGE_NAME\n\n"
+
+# Determine the OS type
+case "$(uname)" in
+    "Linux")
+        MINIFORGE_INSTALLER="Miniforge3-Linux-x86_64.sh"
+        LIBGL_INSTALL=""
+        ;;
+    "Darwin")
+        MINIFORGE_INSTALLER="Miniforge3-MacOSX-x86_64.sh"
+        LIBGL_INSTALL=""
+        export CONDA_SUBDIR=osx-64
+        ;;
+    *)
+        echo "Unsupported OS: $(uname)"
+        exit 1
+        ;;
+esac
+
+# Install Miniforge if it's not already installed
+if ! command -v conda &> /dev/null
+then
+    echo "Miniforge not found, installing Miniforge..."
+
+    # If a MINIFORGE_DIR is not set, ask the user
+    if [ -z "$MINIFORGE_DIR" ]; then
+        MINIFORGE_DIR="$HOME/miniforge3"
+
+        printf "Miniforge will be installed at this location:\n\n" 
+        printf "\t$MINIFORGE_DIR/\n\n"
+        printf "\\n"
+        printf "  - Press ENTER to confirm the Miniforge install location\\n"
+        printf "  - Press CTRL-C to abort the installation\\n"
+        printf "  - Or specify a different location below\\n"
+        printf "\\n"
+        printf "[%s] >>> " "$MINIFORGE_DIR"
+
+        read -r miniforge_install_path
+
+        # If input was given, set it
+        if [ -n "$miniforge_install_path" ]; then
+            MINIFORGE_DIR=$miniforge_install_path
+        fi
+    fi
+
+    curl -kL "https://github.com/conda-forge/miniforge/releases/latest/download/$MINIFORGE_INSTALLER" -o Miniforge3.sh || failed_command "Miniforge download"
+    bash Miniforge3.sh -b -p $MINIFORGE_DIR || failed_command "Miniforge installation"
+    export PATH="$MINIFORGE_DIR/bin:$PATH"
+    mamba init || exit 1
+else
+    echo "Miniforge is already installed."
+    if [[ "$CONDA_PREFIX" == *"envs"* ]]; then
+        MINIFORGE_DIR="${CONDA_PREFIX%/*/*}"
+    else
+        MINIFORGE_DIR=$CONDA_PREFIX
+    fi
+    echo "Setting MINIFORGE_DIR to $MINIFORGE_DIR"
+fi
+
+# If an ENV_NAME was not set, ask the user for it
+if [ -z "$ENV_NAME" ]; then
+    ENV_NAME="isis"
+    printf "ISIS will be installed at this location:\n\n" 
+    printf "\t$MINIFORGE_DIR/envs/$ENV_NAME\n\n"
+    printf "\\n"
+    printf "  - Press ENTER to confirm the ISIS install environment name\\n"
+    printf "  - Press CTRL-C to abort the installation\\n"
+    printf "  - Or specify a different environemnt name below\\n"
+    printf "\\n"
+    printf "[%s] >>> " "$ENV_NAME"
+
+    read -r isis_env_name
+
+    # If a name was given, set it
+    if [ -n "$isis_env_name" ]; then
+        ENV_NAME=$isis_env_name
+    fi
+fi
+
+# Check if the environment already exists 
+if conda env list | grep -qE "^$ENV_NAME[ ]."; then 
+    printf "\nEnvironment \"$ENV_NAME\" already exists. Not performing any updates.\n" 
+    printf "To delete the old environment, use the following commands:\n\n" 
+    printf "\t$ conda deactivate\n" 
+    printf "\t$ conda remove -n $ENV_NAME --all\n\n" 
+else
+    conda config --set ssl_verify false 
```

Comment:
Turning off SSL verification in conda is a bad security practice. I know there are unusual situations where this is necessary, but I don't think it's appropriate to do here, especially without warning a user.

Reply:
Likely a cert issue on my end